src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,49 @@
  */
 
 public class Plane {
+    private String name;
+    private int capacity;
+    private int flightRange;
+    private int power;
+    private int wingspan;
 
+    public Plane(PlaneBuilder planeBuilder) {
+    }
```

Comment:
you passing builder object into constructor and what's next?
also check checklist about all requirements https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html

Reply:
Fixed. Done